app/src/components/common/BalanceRow.vue

```diff
@@ -58,6 +62,11 @@
           <i class="material-icons">send</i></button
         ><span>Send</span>
       </div>
+      <div v-if="stake" class="icon-button-container">
```

Comment:
v-if="!balance.endTime" in line 46 should be `v-if="unstakingBalance"` to make it more readable

Reply:
actually it should be `v-if="!unstakingBalance"`